docs/content/preview/integrations/jaeger.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Jaeger
+linkTitle: Jaeger
+description: Use Jaeger with YCQL API
+menu:
+  preview_integrations:
+    identifier: jaeger
+    parent: integrations
+    weight: 571
+type: docs
+---
+
+Jaeger, inspired by [Dapper](https://research.google.com/pubs/pub36356.html) and [OpenZipkin](http://zipkin.io/), is a distributed tracing system released as open source by [Uber Technologies](http://uber.github.io/). It is used for monitoring and troubleshooting microservices-based distributed systems, including:
+
+- Distributed context propagation
+- Distributed transaction monitoring
+- Root cause analysis
+- Service dependency analysis
+- Performance / latency optimization
+
+Jaeger can be used with various storage backends. YugabyteDB's YCQL API can also be used as a storage backend for Jaeger (version 1.43.0, previous versions are unsupported).
+
+The Jaeger example on this page uses the [HotROD](https://www.jaegertracing.io/docs/1.43/getting-started/#sample-app-hotrod) sample application to illustrate the use of YCQL as a storage backend.
+
+Additionally, it uses Jaeger [All-in-one](https://www.jaegertracing.io/docs/1.43/deployment/#all-in-one), which is a special distribution that combines three Jaeger components, agent, collector, and query service/UI, in a single binary or container image.
+
+{{< note title = "Note">}}
+
+The following example is for testing purposes only. For production deployment, refer to the [Jaeger documentation](https://www.jaegertracing.io/docs/1.18/).
+
+{{< /note >}}
+
+## Prerequisites
+
+To use Jaeger, ensure that you have the following:
+
+- YugabyteDB up and running. Download and install YugabyteDB by following the steps in [Quick start](../../quick-start/).
+
+- [Go 1.20](https://go.dev/doc/go1.20).
+
+## Build the application
+
+To build the application, do the following:
+
+1. Jaeger natively supports two databases, Cassandra and Elasticsearch. So you need to set an environment variable `SPAN_STORAGE_TYPE` to specify your choice of database. For this example, set it to cassandra as follows:
```

Comment:
Can we rephrase `For this example, set it to cassandra as follows:` because we need to set  `SPAN_STORAGE_TYPE` to Cassandra always if we are using YCQL as a backend storage, not only for this example.